main.go

```diff
@@ -75,25 +76,42 @@ func main() {
 			Aliases: []string{"i"},
 			Usage:   "The install strategy allows to install the engine and its dependencies.",
 			Action: func(c *cli.Context) error {
-				fmt.Println("TODO")
+				var context = install.Context{
+					Version:     c.StringSlice("version"),
+					Project:     c.String("project"),
+					Environment: c.String("environment"),
+				}
+
+				for _, engineName := range c.StringSlice("engine") {
+					for _, engine := range catalog.Get() {
+						if engine.Meta.Name == engineName || engine.Meta.ID == engineName {
+							context.Engine = append(context.Engine)
+						}
+					}
+				}
+
+				res, _ := json.MarshalIndent(install.Run(context), "", "    ")
+				fmt.Println(string(res))
+
 				return nil
 			},
 			Flags: []cli.Flag{
-				cli.StringFlag{
+				cli.StringSliceFlag{
 					Name:  "engine,e",
 					Usage: "Engine name to install",
 				},
-				cli.StringFlag{
+				cli.StringSliceFlag{
 					Name:  "version,v",
 					Usage: "Engine version (latest version by default)",
 				},
 				cli.StringFlag{
```

Comment:
`Folder`, `dir` - I think we should rename projects elsewhere in lhman, coz it confuses.